src/Benda95280/MyEntities/MyEntities.php

```diff
@@ -247,11 +332,20 @@ public static function createSkin($skinName)
      * @param $array
      * @param String $arrayname
      * @return CompoundTag
+     * @throws \RuntimeException
+     * @throws \InvalidArgumentException
      */
     public static function arrayToCompTag($array, String $arrayname)
     {
-        $nbt = new BigEndianNBTStream();
-        $tag = $nbt::fromArray($array);
+        $tag = new CompoundTag($arrayname, []);
+        foreach ($array as $key => $value) {
+            if (is_int($value)) $tag->setTag(new IntTag($key, $value));
+            else if (is_string($value)) $tag->setTag(new StringTag($key, $value));
+            else if (is_array($value)) $tag->setTag(MyEntities::arrayToCompTag($value, $key));
+        }
+        return $tag;
+        //This sadly does not work correctly (because of weird property issue) #blamepmmp
```

Comment:
Should we open an Issue on PMMP ?

Reply:
No, it is intentionally done this way, dynamic property access is not allowed - i think 4.0.0 does not have that issue anymore though